internal/config/config.go

```diff
@@ -393,6 +411,45 @@ func oathAuthListHookFunc() mapstructure.DecodeHookFuncType {
 	}
 }
 
+// oauthAuthListHookFunc returns a DecodeHookFunc that converts
+// strings to string slices, when the target type is DotSeparatedStringList.
+// when passed as env should be in the following format:
+// "registry;registry1".
+func gcpAuthListHookFunc() mapstructure.DecodeHookFuncType {
```

Comment:
We need to add the new ENV variable to the README.md: https://github.com/falcosecurity/falcoctl#falcoctl-environment-variables.